pkg/client/blocks.go

```diff
@@ -61,19 +61,43 @@ func (a *Blocks) HeightBySignature(ctx context.Context, id string) (*BlocksHeigh
 	return out, response, nil
 }
 
+func (a *Blocks) HeightByID(ctx context.Context, id proto.BlockID) (*BlocksHeight, *Response, error) {
+	url, err := joinUrl(a.options.BaseUrl, fmt.Sprintf("/blocks/height/%s", id.String()))
+	if err != nil {
+		return nil, nil, err
+	}
+	req, err := http.NewRequest("GET", url.String(), nil)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	out := new(BlocksHeight)
+	response, err := doHttp(ctx, a.options, req, out)
+	if err != nil {
+		return nil, response, err
+	}
+
+	return out, response, nil
+}
+
 type Headers struct {
-	Version          uint64           `json:"version"`
-	Timestamp        uint64           `json:"timestamp"`
-	Reference        proto.BlockID    `json:"reference"`
-	NxtConsensus     NxtConsensus     `json:"nxt-consensus"`
-	Features         []uint64         `json:"features"`
-	Generator        proto.Address    `json:"generator"`
-	Signature        crypto.Signature `json:"signature"`
-	Blocksize        uint64           `json:"blocksize"`
-	TransactionCount uint64           `json:"transactionCount"`
-	Height           uint64           `json:"height"`
-
-	ID proto.BlockID `json:"id"`
+	Version            uint64           `json:"version"`
```

Comment:
Shouldn't we put `omitempty` on new fields here in order to support compatibility with previous version of block.